sections/collections__filter-custom.liquid

```diff
@@ -56,18 +57,19 @@
                         <p>Back</p>
```

Comment:
still seeing a blank square next to the back text on mobile

Reply:
Gotcha. I'll download a png for that as well.